checkout/rust/delivery-customization/default/src/api.rs

```diff
@@ -0,0 +1,48 @@
+use serde::{Deserialize, Serialize};
+
+pub type ID = String;
+
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Input {
+    pub delivery_options: Vec<DeliveryOption>,
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct DeliveryOption {
+    pub id: ID,
+    pub name: String,
+}
+
+#[derive(Clone, Debug, Serialize)]
+pub struct FunctionResult {
+    pub operations: Vec<Operation>,
+}
+
+#[derive(Clone, Debug, Serialize)]
+pub struct Operation {
+    pub hide: Option<HideOperation>,
+    pub remove: Option<MoveOperation>,
```

Comment:
Ah I just remembered, I think it's supposed to be `r#move` because when I tried to do `move`, it complains since it's a keyword in rust. `remove` and `hide` are the same thing, so it makes why Jackson put `r#move`.